### PR TITLE
design static pages

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- ticket redmine: https://edu-redmine.sun-asterisk.vn/issues/75111

## WHAT (optional)
- design static pages, config multiple languages by i18n

## HOW
- create .pug files in views folder and create {{lng}}.json files in locales folder

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: src/views/admin/layoutAdmin.pug [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    link(href="/lib/lightbox/css/lightbox.min.css", rel="stylesheet")
+    link(href="/lib/owlcarousel/assets/owl.carousel.min.css", rel="stylesheet")
+
+    link(href="/css/bootstrap.min.css", rel="stylesheet")
+
+    link(rel='stylesheet' , href='/css/style.css' )
+
+    script(src='https://ajax.googleapis.com/ajax/libs/jquery/3.6.4/jquery.min.js')
+    script(src='https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/js/bootstrap.bundle.min.js')
```
Comment (huongnt-2545) [inline src/views/admin/layoutAdmin.pug]: e tải source về đặt static ở folder public nhé

File: src/views/product/index.pug [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                    .d-flex.mb-2
+                      each _ in [1...product.ratingAvg]
+                        i.fa.fa-star.text-secondary
+                      each _ in [1...(5 - product.ratingAvg)]
+                        i.fa.fa-star
+                    .d-flex.mb-2
+                      h5.fw-bold.me-2 #{product.price} $
+              .d-flex.justify-content-center.my-4
+                a.btn.border.border-secondary.px-4.py-3.rounded-pill.text-primary.w-100(href='#') #{t('product.viewMore')}
+
```
Comment (huongnt-2545) [inline src/views/product/index.pug]: end file

**Comment by ghost** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**